pkgs/by-name/sc/scion/package.nix

```diff
@@ -41,6 +41,13 @@ buildGo121Module {
     ${skipTest 1 "TestOpensslCompatible" "scion-pki/trcs/sign_test.go"}
   '';
 
+  postInstall = ''
+    ln -s $out/bin/dispatcher $out/bin/scion-dispatcher
+    ln -s $out/bin/router $out/bin/scion-router
+    ln -s $out/bin/control $out/bin/scion-control
+    ln -s $out/bin/daemon $out/bin/scion-daemon
```

Comment:
These original names look way too generic. Are they even meant to be put onto `PATH`? Should we hide them?

Reply:
Yeah, good call, we should simply remove them in favor of the less generic names, to make sure nothing has a problem in the future.